modules/apps/transfer/ibc_module_test.go

```diff
@@ -85,14 +90,16 @@ func (suite *TransferTestSuite) TestOnChanOpenInit() {
 
 			tc.malleate() // explicitly change fields in channel and testChannel
 
-			err = cbs.OnChanOpenInit(suite.chainA.GetContext(), channel.Ordering, channel.GetConnectionHops(),
+			version, err := cbs.OnChanOpenInit(suite.chainA.GetContext(), channel.Ordering, channel.GetConnectionHops(),
```

Comment:
Change this to call the transfer IBCModule `OnChanOpenInit` directly. Currently this is getting the top level of the stack (ics29). It is better for us to just test ics20 in this unit test

We can either add a [`IBCTransferModule`](https://github.com/cosmos/ibc-go/blob/main/testing/simapp/app.go#L224) to simapp or call `transfer.NewIBCModule`

```go
ibcModule := transfer.NewIBCModule(suite.chainA.GetSimApp().TransferKeeper)
ibcModule.OnChanOpenInit(...)
```

Reply:
This satisfies the success cases, but oddly fails on the test case for `capability already claimed`. Currently returning with an error:

`cannot claim nil capability: provided capability is nil`. 

Looking into it.